package.json

```diff
@@ -32,18 +32,20 @@
   "dependencies": {
     "babel-preset-es2015": "~6.24.0",
     "babel-register": "~6.26.0",
-    "chai": "~4.1.0",
+    "chai": "~4.1.2",
+    "cucumber": "~3.0.1",
```

Comment:
why do you have cucumber as dependency defined here? It is already a dependency of wdio-cucumber-framework.